### PR TITLE
feat (HIP-423): Long Term Scheduled Transactions

### PR DESCRIPTION
**Pull request by 0xivanov**
Implements HIP-423:
1. Integration tests
2. Example

Fixes: #1165

### CODE REVIEW

File: schedule_create_transaction_e2e_test.go [hunk 1]
```diff
@@ -106,630 +107,900 @@ func TestIntegrationScheduleCreateTransactionCanExecute(t *testing.T) {
...
+
+	info, err := NewScheduleInfoQuery().
+		SetNodeAccountIDs(env.NodeAccountIDs).
+		SetScheduleID(scheduleID).
+		Execute(env.Client)
+	require.NoError(t, err)
+	assert.NotNil(t, info.ExecutedAt)
+}
+
+// TODO
```
Comment (0xivanov) [inline schedule_create_transaction_e2e_test.go:408]: TODO

**Review by mishomihov00 — APPROVED** [review]
Review applies to `.github/workflows/build.yml`

File: schedule_create_transaction_e2e_test.go [hunk 2]
```diff
@@ -106,630 +107,919 @@ func TestIntegrationScheduleCreateTransactionCanExecute(t *testing.T) {
...
+		Execute(env.Client)
+	require.NoError(t, err)
+	require.Nil(t, scheduleInfo.ExecutedAt)
+
+	accountBalanceBefore, err := NewAccountBalanceQuery().
+		SetAccountID(accountId).
+		Execute(env.Client)
+	require.NoError(t, err)
+
+	time.Sleep(10 * time.Second)
```
Comment (gsstoykov) [inline schedule_create_transaction_e2e_test.go:777]: nit: Too much IMHO. Could be validated with less seconds and won't slow CI that much.

File: schedule_create_transaction_e2e_test.go [hunk 3]
```diff
@@ -106,630 +107,919 @@ func TestIntegrationScheduleCreateTransactionCanExecute(t *testing.T) {
...
+	require.NoError(t, err)
+
+	key2, err := GeneratePrivateKey()
+	require.NoError(t, err)
+
+	key3, err := GeneratePrivateKey()
+	require.NoError(t, err)
+
+	key4, err := GeneratePrivateKey()
+	require.NoError(t, err)
```
Comment (gsstoykov) [inline schedule_create_transaction_e2e_test.go:805]: nit: Different convention from the other test where Key list data is populated via loop. 
note: For future and past tests where we need to generate an array of keys maybe we can have something like `generateKeys` for populating KeyLists.

**Review by gsstoykov — COMMENTED** [review]
LG. Just some nits.

**Review by gsstoykov — APPROVED** [review]
LGTM.

**Pull request merged**